picard/script.py

```diff
@@ -967,3 +1104,10 @@ def func_is_video(parser):
 register_script_function(func_title, "title")
 register_script_function(func_is_audio, "is_audio")
 register_script_function(func_is_video, "is_video")
+register_script_function(func_find_str, "find")
+register_script_function(func_reverse_str, "reverse")
+register_script_function(func_substr, "substr")
+register_script_function(func_get_multi_item, "get_multi_item", eval_args=False)
```

Comment:
Could we just call it `get_multi` or maybe `get_item`?

Reply:
`get_multi` has my preference, and we can rename `func_*_multi_item` to `func_*_multi` as well.

Reply:
Changed to $get_multi and func_get_multi.